pkg/apis/performance/v1alpha1/performanceprofile_types.go

```diff
@@ -12,7 +12,17 @@ type PerformanceProfileSpec struct {
 	CPU *CPU `json:"cpu,omitempty"`
 	// HugePages defines set of huge pages related parameters.
 	HugePages *HugePages `json:"hugepages,omitempty"`
-	// NodeSelector is a selector which must be true for the performance profile to fit on a node.
+	// MachineConfigLabel defines the label to add to the MachineConfigs the operator creates. It has to be
+	// used in the MachineConfigSelector of the MachineConfigPool which targets this performance profile.
+	MachineConfigLabel map[string]string `json:"machineConfigLabel,omitempty"`
```

Comment:
can we mark MachineConfigLabel and MachineConfigPoolSelector as both being optional.

Reply:
done :+1: